plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -57,6 +57,12 @@ func ComputeTaintTolerationPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *
 	if node == nil {
 		return schedulerapi.HostPriority{}, fmt.Errorf("node not found")
 	}
+	// get taints from node info cache.
+	taints, err := nodeInfo.Taints()
+	if err != nil {
+		// fallback to computing taints from node Spec
+		taints = node.Spec.Taints
```

Comment:
Yeah, so kubelet updates the node information. I am not sure if we can measure it with local-up. Unless we test at scale, we cannot measure it. I ran kubemark for similar issue sometime back and collected data from kube scheduler log. I can try doing the same for this over the next week.